src/config.example.ts

```diff
@@ -1,8 +1,8 @@
 // #region bot token
-export const TOKEN = 'YOUR_BOT_TOKEN_HERE';
+export const TOKEN = process.env.TOKEN ? process.env.TOKEN : 'YOUR_BOT_TOKEN_HERE';
 // #endregion
 
-export const CLIENT_ID = 'YOUR_CLIENT_ID_HERE';
+export const CLIENT_ID = '';
```

Comment:
I got a build error if this did not exist, added it. Might want to delete if it has a purpose